src/extension.ts

```diff
@@ -72,9 +73,24 @@ export function activate(context: vscode.ExtensionContext) {
 
     context.subscriptions.push(disposable);
 
-    disposable = vscode.commands.registerCommand('extension.ccUpdateDir', () => {
+    disposable = vscode.commands.registerCommand('extension.ccUpdateView', () => {
         if ( vscode.workspace.rootPath )
-            updatePath(vscode.workspace.rootPath);
+            updateObject(vscode.workspace.rootPath);
```

Comment:
To summarize we could:

1. Rename this command "Update Workspace"
2. Optionally add a new command "Update View" that calls "cleartool update" without parameters.
3. Remove the command "Update Directory" and see if we can add it on the tree view context menu.
4. Keep the "Update Current File" as global command.
5. Optionally ad an "Update File" to the context menu as well

what do you think?